LibTCPServer.emx

```diff
@@ -464,22 +469,13 @@ srv->start();
 	    }
 
 	    // Is a port index specified?
-	    int portIndex = -1;
-	    try {
-	    	Var var_portIndex = object->get("portIndex");
-	    	if (!var_portIndex.isEmpty())
-	    		portIndex = var_portIndex.convert<int>();
-	    }
-	    catch (Poco::Exception e) {
-	    	conn->replyStatus(ITCPConnection::Status::Error, "Illegal portIndex: " + e.message());
-	    	return;
-	    }
+	    int portIndex = jsonResult["portIndex"].get_int();
 	    
 	    if (cmd == "sendEvent") {	    
-	    	Var var_priority = object->get("priority");
+	        std::string priorityString =  jsonResult["priority"].get_string();
```

Comment:
Missing error handling. What if the user doesn't specify "priority" as a string?

Reply:
It is handled in below code.